server/src/main/java/org/apache/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -19,39 +19,103 @@
 
 package org.apache.druid.server.coordinator.rules;
 
-import org.apache.druid.java.util.common.IAE;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import org.apache.druid.client.DruidServer;
+import org.apache.druid.common.config.Configs;
+import org.apache.druid.error.InvalidInput;
 import org.apache.druid.timeline.DataSegment;
 
 import java.util.Map;
+import java.util.Objects;
 
 /**
  * LoadRules indicate the number of replicants a segment should have in a given tier.
```

Comment:
Could use a javadoc update with the new property addition.

Reply:
Added doc to the parameter as well. I think `validateTieredReplicants` already covers it.